docs/plugins/discovery-ec2.asciidoc

```diff
@@ -46,6 +46,11 @@ Those that must be stored in the keystore are marked as `Secure`.
 
     An s3 secret key. The `access_key` setting must also be specified. (Secure)
 
+`session_token`::
+
+    An s3 session token. The `access_key` and `secret_key` settings must also
```

Comment:
Nit: An -> A

Reply:
Did I just correct an English about his English grammar? Yes I did. Was I right? Absolutly not.

Thanks :)